src/main/java/com/xjeffrose/chicago/DBManager.java

```diff
@@ -244,6 +245,11 @@ public void destroy() {
         i.next();
       }
 
+      i.seekToLast();
+
+      bb.writeBytes(delimeter.getBytes());
+      bb.writeBytes(i.key());
```

Comment:
we need to add some sort of terminating character to signify the end of the message